internal/convert.go

```diff
@@ -185,6 +185,12 @@ type Audit struct {
 	SkipMetricsPopulation    bool                                   `json:"-"` // Flag to identify if outgoing metrics metadata needs to skipped
 }
 
+// Stores information related to resources.
+type JobIdAndGcloudCmd struct {
+	JobId     string `json:"JobId"`
+	GcloudCmd string `json:"GcloudCmd"`
+}
+
```

Comment:
I think we should rename this to something more generic to allow for future `shard + dataflow` specific stuff to live here

Reply:
Switched to ShardedDataflowJobResources for the lack of a better word.